.github/workflows/notify-non-core-qn.yml

```diff
@@ -0,0 +1,55 @@
+name: Notify Core Team on Non-Core Questions
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  notify_core_team:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check and Notify
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const commenter = context.payload.comment.user.login;
+            const { data: orgMembers } = await github.orgs.listMembers({
+              org: context.repo.owner,
+              role: 'all'
+            });
+            const isOrgMember = orgMembers.some(member => member.login === commenter);
```

Comment:
This should be an array of usernames that we set as a variable.

Reply:
updated.